pkg/volume/util/resize_util.go

```diff
@@ -152,11 +152,12 @@ func MarkControllerReisizeInProgress(pvc *v1.PersistentVolumeClaim, resizerName
 		Status:             v1.ConditionTrue,
 		LastTransitionTime: metav1.Now(),
 	}
-	controllerExpansionInProgress := v1.PersistentVolumeClaimControllerExpansionInProgress
 	conditions := []v1.PersistentVolumeClaimCondition{progressCondition}
 	newPVC := pvc.DeepCopy()
 	newPVC = MergeResizeConditionOnPVC(newPVC, conditions)
-	newPVC.Status.ResizeStatus = &controllerExpansionInProgress
+	newPVC.Status.AllocatedResourceStatus = map[v1.ResourceName]v1.ClaimResourceStatus{
```

Comment:
This may not be the only controller that updates status. You'll need to check if the map exists first. Ditto below

Reply:
fixed.